<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_2{
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}

QLabel{
	color: rgb(80, 80, 80);
	font: 11pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}
QLabel:disabled{
	color: rgb(255, 0, 0);
}
QRadioButton{
	font: 11pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}


QLineEdit{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(235, 235, 235);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}

QLineEdit::hover{
	border-bottom: 2px solid;
	border-bottom-color: rgb(225, 225, 225);
}

QLineEdit:focus{
	border-bottom-color: rgb(243, 116, 53);
}

QLineEdit:disabled{
	background-color: rgb(230, 230, 230);
}



</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="proLabel">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Proprietário(a) *</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="proLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	padding-right: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="farmLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Fazenda *</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="farmLineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="talLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Talhão *</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="talLineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QRadioButton" name="anoRadioButton">
               <property name="styleSheet">
                <string notr="true">QRadioButton{
	border : none;
	color: rgb(80, 80, 80);
}
QRadioButton::indicator{
	border : 1px solid  rgb(80, 80, 80);
	width : 15px;
	height : 15px;
	border-radius :7.5px;
}
QRadioButton::indicator:checked{
	background-color: rgb(243, 116, 53);
	border : 1px solid  rgb(243, 116, 53);
}</string>
               </property>
               <property name="text">
                <string>Ano</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="bordRadioButton">
               <property name="styleSheet">
                <string notr="true">QRadioButton{
	border : none;
	color: rgb(80, 80, 80);
}
QRadioButton::indicator{
	border : 1px solid  rgb(80, 80, 80);
	width : 15px;
	height : 15px;
	border-radius :7.5px;
}
QRadioButton::indicator:checked{
	background-color: rgb(243, 116, 53);
	border : 1px solid  rgb(243, 116, 53);
}</string>
               </property>
               <property name="text">
                <string>Bordadura...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page">
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="anoLabel">
                    <property name="font">
                     <font>
                      <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
                      <pointsize>11</pointsize>
                      <weight>50</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Ano *</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="anoLineEdit">
                    <property name="styleSheet">
                     <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="culLabel">
                    <property name="font">
                     <font>
                      <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
                      <pointsize>11</pointsize>
                      <weight>50</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Cultura *</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="culLineEdit">
                    <property name="styleSheet">
                     <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="appTyLabel">
                    <property name="font">
                     <font>
                      <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
                      <pointsize>11</pointsize>
                      <weight>50</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Tipo de Aplicação *</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="appTyLineEdit">
                    <property name="styleSheet">
                     <string notr="true">QLineEdit:disabled{
	border: none;
	border-bottom: 1px solid;
	border-bottom-color: rgb(255, 0, 0);
	background-color: rgb(250, 250, 250);
	font: 10pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <layout class="QFormLayout" name="formLayout_2">
                  <property name="horizontalSpacing">
                   <number>10</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>20</number>
                  </property>
                  <property name="topMargin">
                   <number>20</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="bordRadioButtonEdit">
                    <property name="styleSheet">
                     <string notr="true">QRadioButton{
	border : none;
	color: rgb(80, 80, 80);
}
QRadioButton::indicator{
	border : 1px solid  rgb(80, 80, 80);
	width : 15px;
	height : 15px;
	border-radius :7.5px;
}
QRadioButton::indicator:checked{
	background-color: rgb(243, 116, 53);
	border : 1px solid  rgb(243, 116, 53);
}</string>
                    </property>
                    <property name="text">
                     <string>Bordadura *</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="bordLineEdit">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="lineRadioButton">
                    <property name="styleSheet">
                     <string notr="true">QRadioButton{
	border : none;
	color: rgb(80, 80, 80);
}
QRadioButton::indicator{
	border : 1px solid  rgb(80, 80, 80);
	width : 15px;
	height : 15px;
	border-radius :7.5px;
}
QRadioButton::indicator:checked{
	background-color: rgb(243, 116, 53);
	border : 1px solid  rgb(243, 116, 53);
}</string>
                    </property>
                    <property name="text">
                     <string>Linhas</string>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineLineEdit">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QRadioButton" name="pointRadioButton">
                    <property name="styleSheet">
                     <string notr="true">QRadioButton{
	border : none;
	color: rgb(80, 80, 80);
}
QRadioButton::indicator{
	border : 1px solid  rgb(80, 80, 80);
	width : 15px;
	height : 15px;
	border-radius :7.5px;
}
QRadioButton::indicator:checked{
	background-color: rgb(243, 116, 53);
	border : 1px solid  rgb(243, 116, 53);
}</string>
                    </property>
                    <property name="text">
                     <string>Pontos</string>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="pointLineEdit">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="cancelPushButton">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	border: none;
	color: rgb(120, 120, 120);
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}

QPushButton::hover{
	color: rgb(100, 100, 100);
}
</string>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addPushButton">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(243, 116, 53);
	border: none;	
	color: rgb(250, 250, 250);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
}</string>
             </property>
             <property name="text">
              <string>Adicionar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
